### ISSUE TITLE
Failed when installing

### ISSUE THREAD

**Issue by dotnetwise** [meteor/windows-preview#92]
Does not install on Windows Server 2012 / Azure
```
[51CC:3F08][2015-03-10T15:03:56]i299: Plan complete, result: 0x0
[51CC:3F08][2015-03-10T15:03:56]i300: Apply begin
[51CC:3F08][2015-03-10T15:03:56]i000: Caching bundle from: 'C:\Users\DEVELO~1\AppData\Local\Temp\2\{1474e220-151a-45a3-86b8-b3ee16c36bb8}\.be\Setup_Meteor.exe' to: 'C:\Users\developer\AppData\Local\Package Cache\{1474e220-151a-45a3-86b8-b3ee16c36bb8}\Setup_Meteor.exe'
[51CC:3F08][2015-03-10T15:03:56]i320: Registering bundle dependency provider: {1474e220-151a-45a3-86b8-b3ee16c36bb8}, version: 0.2.0.0
[51CC:24E4][2015-03-10T15:03:56]i305: Verified acquired payload: SetupMeteor_UserInstall at path: C:\Users\developer\AppData\Local\Package Cache\.unverified\SetupMeteor_UserInstall, moving to: C:\Users\developer\AppData\Local\Package Cache\{D6252E9C-67B6-431C-95B7-F8DD8A1FB48B}v0.2.0\SetupMeteor.msi.
[51CC:3F08][2015-03-10T15:03:56]i323: Registering package dependency provider: {D6252E9C-67B6-431C-95B7-F8DD8A1FB48B}, version: 0.2.0, package: SetupMeteor_UserInstall
[51CC:3F08][2015-03-10T15:03:56]i301: Applying execute package: SetupMeteor_UserInstall, action: Install, path: C:\Users\developer\AppData\Local\Package Cache\{D6252E9C-67B6-431C-95B7-F8DD8A1FB48B}v0.2.0\SetupMeteor.msi, arguments: ' ARPSYSTEMCOMPONENT="1" MSIFASTINSTALL="7" METEOR_DIR="C:\Users\developer\AppData\Local\" INSTALL_REG_PATH="SOFTWARE\Meteor Development Group \Meteor\Install" MSICUSTOMERRFILE="C:\Users\DEVELO~1\AppData\Local\Temp\Meteor_20150310150354.log.err" PERUSER_INSTALL="1"'
[51CC:3F08][2015-03-10T15:03:57]e000: Error 0x80070659: Failed to install MSI package.
[51CC:3F08][2015-03-10T15:03:57]e000: Error 0x80070659: Failed to configure per-user MSI package.
[51CC:3F08][2015-03-10T15:03:57]i319: Applied execute package: SetupMeteor_UserInstall, result: 0x80070659, restart: None
[51CC:3F08][2015-03-10T15:03:57]e000: Error 0x80070659: Failed to execute MSI package.
[51CC:3F08][2015-03-10T15:03:57]i318: Skipped rollback of package: SetupMeteor_UserInstall, action: Uninstall, already: Absent
[51CC:3F08][2015-03-10T15:03:57]i319: Applied rollback package: SetupMeteor_UserInstall, result: 0x0, restart: None
[51CC:3F08][2015-03-10T15:03:57]i329: Removed package dependency provider: {D6252E9C-67B6-431C-95B7-F8DD8A1FB48B}, package: SetupMeteor_UserInstall
[51CC:3F08][2015-03-10T15:03:57]i351: Removing cached package: SetupMeteor_UserInstall, from path: C:\Users\developer\AppData\Local\Package Cache\{D6252E9C-67B6-431C-95B7-F8DD8A1FB48B}v0.2.0\
[51CC:3F08][2015-03-10T15:03:57]i330: Removed bundle dependency provider: {1474e220-151a-45a3-86b8-b3ee16c36bb8}
[51CC:3F08][2015-03-10T15:03:57]i352: Removing cached bundle: {1474e220-151a-45a3-86b8-b3ee16c36bb8}, from path: C:\Users\developer\AppData\Local\Package Cache\{1474e220-151a-45a3-86b8-b3ee16c36bb8}\
[51CC:3F08][2015-03-10T15:03:57]i399: Apply complete, result: 0x80070659, restart: None, ba requested restart:  No
```

**Comment by stubailo** [comment]
Hi, can you tell me your purpose for using Meteor on Windows Server 2012? Is your personal computer running Windows Server, or are you remoting into a computer to do development?

**Comment by dotnetwise** [comment]
Because it's a build server and from octopus we need to do `meteor deploy`.

**Comment by stubailo** [comment]
Ah, that makes a lot of sense actually.

**Comment by dotnetwise** [comment]
So any luck with this? 
We kinda need it, and The logs seem completly irellevant

**Comment by stubailo** [comment]
Right now the installer doesn't work on Windows Server 2008 or Windows Server 2012. We're looking into fixing it or providing a workaround soon (like by the end of this week)

**Comment by dotnetwise** [comment]
Meteor `0.3.0 rc1`  is still not instaling on Windows Server 2012

Here is an updated log:
```
[46C0:183C][2015-03-17T08:38:33]i001: Burn v3.8.1128.0, Windows v6.3 (Build 9600: Service Pack 0), path: C:\E2ETools\files\InstallMeteor.exe, cmdline: ''
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'TermsUrl' to value 'https://www.meteor.com/terms'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'PrivacyUrl' to value 'https://www.meteor.com/privacy'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing numeric variable 'PerMachineInstall' to value '0'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing numeric variable 'PerUserInstall' to value '1'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing numeric variable 'CreateRButton' to value '1'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing numeric variable 'SignInRButton' to value '0'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing hidden variable 'RegisterEmail'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing hidden variable 'RegisterUser'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing hidden variable 'RegisterPass'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'SkipRegistrationState' to value 'disable'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing numeric variable 'SkipRegistration' to value '0'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'UserMeteorSessionFile' to value '[LocalAppDataFolder]\.meteorsession'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'PerMachineInstallFolder' to value '[ProgramFilesFolder][WixBundleName]'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'PerUserInstallFolder' to value '[LocalAppDataFolder]'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'InstallFolder' to value '[PerUserInstallFolder]'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'InstallRegPath' to value 'SOFTWARE\Meteor Development Group \Meteor\Install'
[46C0:183C][2015-03-17T08:38:33]i000: Initializing string variable 'MSICustomErrFile' to value '[WixBundleLog].err'
[46C0:183C][2015-03-17T08:38:33]i000: Setting string variable 'WixBundleLog' to value 'C:\Users\DEVELO~1\AppData\Local\Temp\Meteor_20150317083833.log'
[46C0:183C][2015-03-17T08:38:33]i000: Setting string variable 'WixBundleOriginalSource' to value 'C:\E2ETools\files\InstallMeteor.exe'
[46C0:183C][2015-03-17T08:38:33]i052: Condition '(VersionNT >= v5.1)' evaluates to true.
[46C0:41C8][2015-03-17T08:38:33]i000: Setting string variable 'varProgressHeader' to value 'Installing [WixBundleName]'
[46C0:41C8][2015-03-17T08:38:33]i000: Setting string variable 'varProgressInfo' to value 'Please wait while Setup installs [WixBundleName] on your computer.'
[46C0:41C8][2015-03-17T08:38:33]i000: Setting string variable 'varSuccessHeader' to value 'Meteor Installed'
[46C0:41C8][2015-03-17T08:38:33]i000: Setting string variable 'varSuccessInfo' to value 'Meteor is now installed on your computer.

Open a new command prompt and run "meteor".'
[46C0:41C8][2015-03-17T08:38:33]i000: Setting string variable 'varFailureHeader' to value 'Setup Failed'
[46C0:183C][2015-03-17T08:38:33]i100: Detect begin, 2 packages
[46C0:183C][2015-03-17T08:38:33]i102: Detected related bundle: {1474e220-151a-45a3-86b8-b3ee16c36bb8}, type: Upgrade, scope: PerUser, version: 0.2.0.0, operation: MajorUpgrade
[46C0:183C][2015-03-17T08:38:33]i000: Update location: https://www.meteor.com/.
[46C0:183C][2015-03-17T08:38:33]i101: Detected package: SetupMeteor_MachineInstall, state: Absent, cached: None
[46C0:183C][2015-03-17T08:38:33]i101: Detected package: SetupMeteor_UserInstall, state: Absent, cached: None
[46C0:183C][2015-03-17T08:38:33]i199: Detect complete, result: 0x0
[46C0:183C][2015-03-17T08:38:33]i200: Plan begin, 2 packages, action: Install
[46C0:183C][2015-03-17T08:38:33]i052: Condition 'NOT Installed AND (PerMachineInstall = 1)' evaluates to false.
[46C0:183C][2015-03-17T08:38:33]w322: Skipping cross-scope dependency registration on package: SetupMeteor_MachineInstall, bundle scope: PerUser, package scope: PerMachine
[46C0:183C][2015-03-17T08:38:33]i052: Condition 'NOT Installed AND (PerUserInstall = 1)' evaluates to true.
[46C0:183C][2015-03-17T08:38:33]i000: Setting string variable 'WixBundleRollbackLog_SetupMeteor_UserInstall' to value 'C:\Users\DEVELO~1\AppData\Local\Temp\Meteor_20150317083833_0_SetupMeteor_UserInstall_rollback.log'
[46C0:183C][2015-03-17T08:38:33]i000: Setting string variable 'WixBundleLog_SetupMeteor_UserInstall' to value 'C:\Users\DEVELO~1\AppData\Local\Temp\Meteor_20150317083833_0_SetupMeteor_UserInstall.log'
[46C0:183C][2015-03-17T08:38:33]i201: Planned package: SetupMeteor_MachineInstall, state: Absent, default requested: Absent, ba requested: Absent, execute: None, rollback: None, cache: No, uncache: No, dependency: None
[46C0:183C][2015-03-17T08:38:33]i201: Planned package: SetupMeteor_UserInstall, state: Absent, default requested: Present, ba requested: Present, execute: Install, rollback: Uninstall, cache: Yes, uncache: No, dependency: Register
[46C0:183C][2015-03-17T08:38:33]i207: Planned related bundle: {1474e220-151a-45a3-86b8-b3ee16c36bb8}, type: Upgrade, default requested: Absent, ba requested: Absent, execute: Uninstall, rollback: Install, dependency: None
[46C0:183C][2015-03-17T08:38:33]i299: Plan complete, result: 0x0
[46C0:183C][2015-03-17T08:38:33]i300: Apply begin
[46C0:183C][2015-03-17T08:38:33]i320: Registering bundle dependency provider: {84a64acd-e2c9-4e51-9361-10d888ee0372}, version: 0.3.0.0
[46C0:4A58][2015-03-17T08:38:33]i305: Verified acquired payload: SetupMeteor_UserInstall at path: C:\Users\developer\AppData\Local\Package Cache\.unverified\SetupMeteor_UserInstall, moving to: C:\Users\developer\AppData\Local\Package Cache\{3984B4E3-3783-491B-9395-5A5958FB7C3D}v0.3.0\SetupMeteor.msi.
[46C0:183C][2015-03-17T08:38:33]i323: Registering package dependency provider: {3984B4E3-3783-491B-9395-5A5958FB7C3D}, version: 0.3.0, package: SetupMeteor_UserInstall
[46C0:183C][2015-03-17T08:38:33]i301: Applying execute package: SetupMeteor_UserInstall, action: Install, path: C:\Users\developer\AppData\Local\Package Cache\{3984B4E3-3783-491B-9395-5A5958FB7C3D}v0.3.0\SetupMeteor.msi, arguments: ' ARPSYSTEMCOMPONENT="1" MSIFASTINSTALL="7" METEOR_DIR="C:\Users\developer\AppData\Local\" INSTALL_REG_PATH="SOFTWARE\Meteor Development Group \Meteor\Install" MSICUSTOMERRFILE="C:\Users\DEVELO~1\AppData\Local\Temp\Meteor_20150317083833.log.err" PERUSER_INSTALL="1"'
[46C0:183C][2015-03-17T08:38:33]e000: Error 0x80070659: Failed to install MSI package.
[46C0:183C][2015-03-17T08:38:33]e000: Error 0x80070659: Failed to configure per-user MSI package.
[46C0:183C][2015-03-17T08:38:33]i319: Applied execute package: SetupMeteor_UserInstall, result: 0x80070659, restart: None
[46C0:183C][2015-03-17T08:38:33]e000: Error 0x80070659: Failed to execute MSI package.
[46C0:183C][2015-03-17T08:38:33]i318: Skipped rollback of package: SetupMeteor_UserInstall, action: Uninstall, already: Absent
[46C0:183C][2015-03-17T08:38:33]i319: Applied rollback package: SetupMeteor_UserInstall, result: 0x0, restart: None
[46C0:183C][2015-03-17T08:38:33]i329: Removed package dependency provider: {3984B4E3-3783-491B-9395-5A5958FB7C3D}, package: SetupMeteor_UserInstall
[46C0:183C][2015-03-17T08:38:33]i351: Removing cached package: SetupMeteor_UserInstall, from path: C:\Users\developer\AppData\Local\Package Cache\{3984B4E3-3783-491B-9395-5A5958FB7C3D}v0.3.0\
[46C0:183C][2015-03-17T08:38:33]i330: Removed bundle dependency provider: {84a64acd-e2c9-4e51-9361-10d888ee0372}
[46C0:183C][2015-03-17T08:38:33]i352: Removing cached bundle: {84a64acd-e2c9-4e51-9361-10d888ee0372}, from path: C:\Users\developer\AppData\Local\Package Cache\{84a64acd-e2c9-4e51-9361-10d888ee0372}\
[46C0:183C][2015-03-17T08:38:33]i399: Apply complete, result: 0x80070659, restart: None, ba requested restart:  No
```

**Comment by dotnetwise** [comment]
Lol, if you explicitely, manually run the installer as administrator, it does work! D'oh!!

**Comment by stubailo** [comment]
Hmm, you shouldn't have to do this. We'll look into it.

**Comment by dotnetwise** [comment]
I have found another issue. I want meteor to run under a different account, but currently it is installed only under the current logged user account.

How can I install it "globally", so any account can run it?

**Comment by stubailo** [comment]
@dotnetwise We currently don't support installing for all user accounts. It's a similar issue to #89

**Comment by dotnetwise** [comment]
Does `0.3.0` contain all the bells and whistles from `1.0.4.1` ?

**Comment by stubailo** [comment]
@dotnetwise I think it does! When it is finally released it will be called 1.1.

**Comment by stubailo** [comment]
OK this is now a duplicate of #125 

Thanks for giving us motivation to fix Windows Server!
